### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by williamrolando88**
Added the following features:

- TailwindCSS
- HTML skeleton
- Project style
- Javascript rendering elements

### CODE REVIEW

File: src/classes.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  constructor(head) {
+    this.head = head;
+    this.size = 0;
+  }
+
+  add(activity) {
+    const task = new Activity(task);
+  }
+}
+*/
```
Comment (Dipeshtwis) [inline src/classes.js:19]: - [ ] I would recommend removing all the commented code to make the codebase cleaner.

File: src/index.html [hunk 2]
```diff
@@ -4,25 +4,52 @@
...
+              type="text"
+              name="todo1"
+              id="todo1"
+            />
+          </div>
+          <div class="flex items-center flex-grow-0">
+            <span class="material-icons"> drag_indicator </span>
+            <span class="material-icons hidden"> delete </span>
+          </div>
+        </li> -->
```
Comment (Dipeshtwis) [inline src/index.html:48]: - [ ] I would recommend removing all the commented code to make the codebase cleaner.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @williamrolando88, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [x] README file should have a title and description about the project even if it's one or two-line so that anyone can understand your project. Also, write the steps to run the project locally.

- [x] In the README file the `live link` is taking me to another website. Kindly fix that to make the README professional or remove the section.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @williamrolando88, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**